### PR TITLE
Add WikibaseQualityConstraints and WikibaseLexeme (T6646)

### PR DESCRIPTION
**Pull request by R4356th**
Adds functionality for installing WikibaseQualityConstraints and WikibaseLexeme. Configuration is a to-do.

### CODE REVIEW

File: ManageWikiExtensions.php [hunk 1]
```diff
@@ -2571,6 +2571,32 @@
...
+			'name' => 'WikibaseQualityConstraints',
+			'linkPage' => 'https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:WikibaseQualityConstraints',
+			'var' => 'wmgUseWikibaseQualityConstraints',
+			'conflicts' => false,
+			'requires' => [],
+			'install' => [
+				'sql' => [
+					'wbqc_create' => "$IP/extensions/WikibaseQualityConstraints/repo/sq/create_wbqc_constraints.sql",
+					'wbqc_constraint_id' => "$IP/extensions/WikibaseQualityConstraints/repo/sq/patch-wbqc_constraints-constraint_id.sql",
+					'wbqc_constraints_guid_uniq' => "$IP/extensions/WikibaseQualityConstraints/repo/sq/patch-wbqc_constraints-wbqc_constraints_guid_uniq.sql",
```
Comment (Universal-Omega) [inline ManageWikiExtensions.php:2584]: using the wrong sql path. Should be `$IP/extensions/WikibaseQualityConstraints/sql/*` not `$IP/extensions/WikibaseQualityConstraints/repo/sq/*`

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiExtensions.php (hunk 1 above)
Comment (R4356th) [reply ManageWikiExtensions.php:2584]: Oops...

File: ManageWikiExtensions.php [hunk 2]
```diff
@@ -2571,6 +2571,32 @@
...
+			'name' => 'WikibaseQualityConstraints',
+			'linkPage' => 'https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:WikibaseQualityConstraints',
+			'var' => 'wmgUseWikibaseQualityConstraints',
+			'conflicts' => false,
+			'requires' => [],
+			'install' => [
+				'sql' => [
+					'wbqc_create' => "$IP/extensions/WikibaseQualityConstraints/repo/sql/create_wbqc_constraints.sql",
+					'wbqc_constraint_id' => "$IP/extensions/WikibaseQualityConstraints/repo/sql/patch-wbqc_constraints-constraint_id.sql",
+					'wbqc_constraints_guid_uniq' => "$IP/extensions/WikibaseQualityConstraints/repo/sql/patch-wbqc_constraints-wbqc_constraints_guid_uniq.sql",
```
Comment (Universal-Omega) [inline ManageWikiExtensions.php:2584]: Still wrong. not `/repo/`

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiExtensions.php (hunk 2 above)
Comment (R4356th) [reply ManageWikiExtensions.php:2584]: I saw it above.

Comment (R4356th) [reply ManageWikiExtensions.php:2584]: Oh, okay. I get it now.

File: ManageWikiExtensions.php [hunk 3]
```diff
@@ -2571,6 +2571,32 @@
...
+			'name' => 'WikibaseQualityConstraints',
+			'linkPage' => 'https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:WikibaseQualityConstraints',
+			'var' => 'wmgUseWikibaseQualityConstraints',
+			'conflicts' => false,
+			'requires' => [],
+			'install' => [
+				'sql' => [
+					'wbqc_create' => "$IP/extensions/WikibaseQualityConstraints/sql/create_wbqc_constraints.sql",
+					'wbqc_constraint_id' => "$IP/extensions/WikibaseQualityConstraints/sql/patch-wbqc_constraints-constraint_id.sql",
+					'wbqc_constraints_guid_uniq' => "$IP/extensions/WikibaseQualityConstraints/sql/patch-wbqc_constraints-wbqc_constraints_guid_uniq.sql",
```
Comment (Universal-Omega) [inline ManageWikiExtensions.php:2584]: You only need the first SQL file, the other ones are just patches

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Pull request merged**